docs/Encapsulation.md

```diff
@@ -47,9 +47,9 @@ fastify.register(async function authenticatedContext (childServer) {
   childServer.register(require('fastify-bearer-auth'), { keys: ['abc123'] })
 
   childServer.route({
-    path: '/one',
+    url: '/one',
     method: 'GET',
-    handler (request, response) {
+    handler: (request, response) => {
```

Comment:
I very specifically did not use an arrow function here. I used shorthand method notation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#description